practico_08/autogestion_alumnos/__init__.py

```diff
@@ -14,6 +15,7 @@ def create_app(config_class):
     app = Flask(__name__)
     app.config.from_object(config_class)
 
+    bootstrap = Bootstrap(app)
```

Comment:
Is it more important that the `init` ? I think that this should be the last in the list of these initializations. Can you change this?